docs/fundamentals/code-analysis/quality-rules/ca1865-ca1867.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "CA1865: Use char overload"
+description: "Use 'string.Method(char)' instead of 'string.Method(string)' when you have a string with a single char"
+ms.date: 07/27/2023
+ms.topic: reference
+f1_keywords:
+  - CA1865
+helpviewer_keywords:
+  - CA1865
+dev_langs:
+  - CSharp
+  - VB
+author: mrahhal
+---
+
+# CA1865-CA1867: Use 'string.Method(char)' instead of 'string.Method(string)' when you have a string with a single char
+
+|                                     | Value                                  |
+| ----------------------------------- | -------------------------------------- |
+| **Rule ID**                         | CA1865-CA1867                          |
+| **Category**                        | [Performance](performance-warnings.md) |
+| **Fix is breaking or non-breaking** | Non-breaking                           |
+| **Enabled by default in .NET 7**    | No                                     |
```

Comment:
```suggestion
Should this be true for 1865 and 1866?
```

Reply:
I believe this is "No" for all the three rules (since they're .net 8 rules). Might need @tarekgh to confirm.

Reply:
I don't mind, but all the other new .net 8 analyzers have this same "Enabled by default in .NET 7" (e.g. [1](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1868), [2](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1864), [3](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1861))

So I feel this has a slight different meaning, and it should remain consistent too.